src/features/claims/claims-list/FilterElements.tsx

```diff
@@ -0,0 +1,134 @@
+import styled from '@emotion/styled'
+import { Checkbox, Flex, Label } from '@hedvig-ui'
+import { useArrowKeyboardNavigation } from '@hedvig-ui/hooks/keyboard/use-arrow-keyboard-navigation'
+import { Radio } from '@hedvig-ui/Radio/radio'
+import { numberMemberGroupsOptions } from 'features/questions/number-member-groups-radio-buttons'
+import React, { useEffect } from 'react'
+import { LabelWithPopover } from './filters/ClaimListFilters'
+
+export const Filters = styled(Flex)<{ active: boolean }>`
+  width: fit-content;
+  border: ${({ active, theme }) =>
+    active ? `2px solid ${theme.accentLight}` : 'none'};
+`
+
+export const FilterElementStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0.7rem;
+
+  & label {
+    margin: 0;
+  }
+`
+
+interface FilterElementProps {
+  active: boolean
+  checked: (key: string) => boolean
+  onChange: (key: string) => void
+  onPerfom: (index: number) => void
+  label: string
+  values: any
+  popover?: string
+  CheckboxLabel?: any
+  getContent: (key: string) => React.ReactNode
```

Comment:
```suggestion
  content: (key: string) => React.ReactNode
```

But also, why not remove this prop and use `children` instead, if we're returning a node anyway?

Reply:
Ah, is it because you want to render conditionally on key? In that case, just rename it to `onRender` 🤓